apps/android/LiveVideoApp/app/src/main/AndroidManifest.xml

```diff
@@ -11,7 +11,8 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:supportsRtl="true"
-        tools:targetApi="31">
+        tools:targetApi="31"
+        android:networkSecurityConfig="@xml/network_security_config">
```

Comment:
added new network security config to the main app, to allow external proxy intercept all the api Calls only on Debug Build.